plugins/azure-devops/README.md

```diff
@@ -16,6 +16,12 @@ Lists the top _n_ Active, Completed, or Abandoned Pull Requests for a given repo
 
 ![Azure Repos Pull Requests Example](./docs/azure-devops-pull-requests.png)
 
+### Azure Git Tags
```

Comment:
This should be "Azure Repos Git Tags" seeing as Git Tags is a feature of Azure Repos